app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -120,21 +117,26 @@ export const Papers: React.FC<IPapersProps> = (props) => {
   const fetch = React.useCallback(
     async (filter: IPaperFilter & Partial<IContentListAdvancedFilter>) => {
       try {
-        setLoading(true);
-        const data = await findContent(
-          makeFilter({
-            ...filter,
-          }),
-        );
-        return new Page(data.page - 1, data.quantity, data?.items, data.total);
+        if (!isLoading) {
```

Comment:
If isLoading == true, the finally section will set it to false right away, is that what expected?